StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/EmbeddedPaymentMethodsViewSnapshotTests.swift

```diff
@@ -0,0 +1,314 @@
+                                                      shouldShowApplePay: true,
+                                                      shouldShowLink: true,
+                                                      savedPaymentMethodAccessoryType: .none)
+
+        verify(embeddedView)
+    }
+
+    func testEmbeddedPaymentMethodsView_flatRadio_bottomSeparatorDisabled() {
+        var appearance: PaymentSheet.Appearance = .default
+        appearance.paymentOptionView.paymentMethodRow.flat.bottomSeparatorEnabled = false
+
+        let embeddedView = EmbeddedPaymentMethodsView(paymentMethodTypes: [.stripe(.card), .stripe(.cashApp)],
+                                                      savedPaymentMethod: nil,
+                                                      appearance: appearance,
+                                                      shouldShowApplePay: true,
+                                                      shouldShowLink: true,
+                                                      savedPaymentMethodAccessoryType: .none)
+
+        verify(embeddedView)
+    }
+
+    func testEmbeddedPaymentMethodsView_flatRadio_colorSelected() {
+        var appearance: PaymentSheet.Appearance = .default
+        appearance.paymentOptionView.paymentMethodRow.flat.radio.colorSelected = .red
+
+        let embeddedView = EmbeddedPaymentMethodsView(paymentMethodTypes: [.stripe(.card), .stripe(.cashApp)],
+                                                      savedPaymentMethod: nil,
+                                                      appearance: appearance,
+                                                      shouldShowApplePay: true,
+                                                      shouldShowLink: true,
+                                                      savedPaymentMethodAccessoryType: .none)
+
+        // Simulate tapping a button
+        if let rowButton = embeddedView.stackView.arrangedSubviews.first(where: { $0 is RowButton }) as? RowButton {
+            embeddedView.handleRowSelection(selectedRowButton: rowButton)
+        }
+
+        verify(embeddedView)
+    }
+
+    func testEmbeddedPaymentMethodsView_flatRadio_genericTestAppearance() {
+        let embeddedView = EmbeddedPaymentMethodsView(paymentMethodTypes: [.stripe(.card), .stripe(.cashApp)],
+                                                      savedPaymentMethod: ._testCard(),
+                                                      appearance: ._testMSPaintTheme,
+                                                      shouldShowApplePay: true,
+                                                      shouldShowLink: true,
+                                                      savedPaymentMethodAccessoryType: .viewMore)
+
+        // Simulate tapping a button
+        if let rowButton = embeddedView.stackView.arrangedSubviews.first(where: { $0 is RowButton }) as? RowButton {
+            embeddedView.handleRowSelection(selectedRowButton: rowButton)
```

Comment:
Does this do anything? I think this taps the saved card or card row

Reply:
It does select the saved PM, nothing is selected by default and is what I believe is expected based on the figmas.
![testEmbeddedPaymentMethodsView_flatRadio_genericTestAppearance@3x](https://github.com/user-attachments/assets/e94c5115-365c-4304-b4b3-9d9f27a3b226)

Reply:
Ah ok. What's selected by default should (eventually) match vertical mode logic. This code might break when the view does what it really should do when you tap the saved PM (open the sheet)?

Reply:
I think the initial selection point is a good one, but tapping the row shouldn't open the sheet IMO, just tapping the "View more" button. I'll update this to select the last row.